heartbeat/monitors/active/icmp/icmp.go

```diff
@@ -71,6 +71,12 @@ func create(
 		}
 
 		settings := monitors.MakeHostJobSettings(jobName, host, config.Mode)
+		metadata, e := monitors.AddMetadata(config.EventMetadata)
```

Comment:
Can you call it `err` instead of `e` to be consistent with most of the other code?